### DISCUSSION TITLE
Exclude a page from search results

### DISCUSSION THREAD

**Discussion by RensTillmann** [GitbookIO/community#137] [Feature Requests]
### What's your feature/idea?

Option to exclude a specific page (or multiple pages) from the search results.

### What's your desired outcome?

When a user uses the search functionality they will only be presented with results from pages that are not on the exclude list.

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

### Anything else?

For me this is critical because I want to provide a full changelog on a page, instead of linking to an external page. However every time you search something this changelog will be at the top of the results. I don't want this. Right now the only solution would be to link to an external source e.g. github or online file stored elsewhere.

**Comment by aikoii** [comment]
Hey @RensTillmann we just shipped this feature 🎉. Check out #730 for more details. Thanks again for your feedback 👋
